main.go

```diff
@@ -160,3 +160,36 @@ func main() {
 		log.Fatalf("HTTP shutdown: %v", err) // nolint:gocritic
 	}
 }
+
+func (e *elizaServer) Introduce(
+	ctx context.Context,
+	req *connect.Request[elizav1.IntroduceRequest],
+	stream *connect.ServerStream[elizav1.IntroduceResponse],
+) error {
+	name := req.Msg.Name
+	if name == "" {
+		return connect.NewError(connect.CodeInvalidArgument, fmt.Errorf(
+			"name is required",
+		))
+	}
+	intros := eliza.GetIntroResponses(name)
+	for _, resp := range intros {
+		if err := stream.Send(&elizav1.IntroduceResponse{Sentence: resp}); err != nil {
+			return err
+		}
+		time.Sleep(500)
```

Comment:
Can we make the sleep time a field on `elizaServer`? I'd like to keep the unit tests fast, but sleep for a noticeable length of time in production.

Also, this is 500 _nanoseconds_ - it's too short to be useful. MacOS, for example, coalesces timers into 15ms windows.